e2e/pages/caseList.page.js

```diff
@@ -53,7 +54,7 @@ module.exports = {
   },
 
   locateCase(caseId){
-    return locate(`//ccd-search-result/table//tr[//a[contains(@href,'${caseId}')]]`);
+    return `a[href$='${caseId}']`;
```

Comment:
Not 100% up to date on this notation but I assume it is still saying a link whose href contains the case number

Reply:
the difference is that first is xpath second is css selector
waitForSelector does now work with xpath selectors

Reply:
ahh it means href that ends in case id, makes sense